pkg/arvo/lib/base64.hoon

```diff
@@ -128,6 +128,6 @@
     =/  len  (sub (mul 3 (div (add lat dif) 4)) dif)
     :+  ~  len
     %+  swp  3
-    (repn 6 (flop (weld dat (reap dif 0))))
+    (rep 0 6 (flop (weld dat (reap dif 0))))
```

Comment:
This one seems particularly egregious wrt "just do all these operations in atom-space",
`(repn 0 6 (flop (weld dat (reap dif 0)))` is I think the same as `(lsh 0 dif (repn 0 6 (flop dat)))`.

Which granted seems to still has a 6-bit swap followed by an 8-bit swap so I'm not sure how much can be done…

Reply:
(Honestly we should be doing our _parsing_ in atom-and-offset space, wherein the string slice could be simd converted to bytes-of-u6 and then squeezed down from "bloq" 6 bits to bloq bytes in a single jetted operation, but that's a whole other can of worms.)

Reply:
`(lsh 0 (mul 6 dif) ...)`, I think. I completely agree wrt parsing.

Reply:
`(lsh  .2.585 dif ...)`, quite